src/helpers/limit_results.js

```diff
@@ -0,0 +1,8 @@
+const limitResults = (results) => {
+  while (results.length != 9 && results.length > 9) {
```

Comment:
Because I wanted to limit the results of how many startups we are getting by 9, but i guess from the last phone call that we had you wanted it to be more than 9 right ? and how much ? @esraajb